concepticondata/conceptlists/Nagano-2013-1256.tsv

```diff
@@ -0,0 +1,1257 @@
+Nagano-2013-1256-548	548	1003	touch (v)	1892	TOUCH	
+Nagano-2013-1256-549	549	1005	rub (v)	1449	RUB	
+Nagano-2013-1256-550	550	1007	wave (v)			
+Nagano-2013-1256-551	551	1009	shake (v)	1909	SHAKE	
+Nagano-2013-1256-552	552	1010	stir (v)	696	STIR	
+Nagano-2013-1256-553	553	1011	earthquake	6	EARTHQUAKE	
+Nagano-2013-1256-554	554	1013	push (vt)	1452	PUSH	
+Nagano-2013-1256-555	555	1015	press (vt)	1913	PRESS	
+Nagano-2013-1256-556	556	1016	oppress (vt)	401	OPPRESS	
+Nagano-2013-1256-557	557	1018	pull (vt)	1455	PULL	
+Nagano-2013-1256-558	558	1020	lengthen (vt)			
+Nagano-2013-1256-559	559	1021	postpone (v)			
+Nagano-2013-1256-560	560	1023	squeeze (v)	1414	SQUEEZE	
+Nagano-2013-1256-561	561	1024	buckle, tie (v)	1917	TIE	
+Nagano-2013-1256-562	562	1026	carry on the back (v)	465	CARRY ON BACK	
+Nagano-2013-1256-563	563	1028	kick (v)	1135	KICK	
+Nagano-2013-1256-564	564	1030	tread on it			
+Nagano-2013-1256-565	565	1032	use (v)	2072	USE (VERB)	
+Nagano-2013-1256-566	566	1033	useful			
+Nagano-2013-1256-567	567	1035	lose it (v)	1660	LOSE	
+Nagano-2013-1256-568	568	1037	hide it (v)	2486	HIDE	
+Nagano-2013-1256-569	569	1038	cover, conceal (v)	602	HIDE (CONCEAL)	
+Nagano-2013-1256-570	570	1040	put it away (v)			
+Nagano-2013-1256-571	571	1041	think a great deal of (v)			
+Nagano-2013-1256-572	572	1043	seek it (v)	1468	LOOK FOR	
+Nagano-2013-1256-573	573	1045	find it (v)	719	FIND	
+Nagano-2013-1256-574	574	1047	show it (v)	1039	SHOW	
+Nagano-2013-1256-575	575	1049	put it (v)	998	PUT	
+Nagano-2013-1256-576	576	1051	wrap it (v)	1113	WRAP	
+Nagano-2013-1256-577	577	1053	spread it out (v)	38	SPREAD OUT	
+Nagano-2013-1256-578	578	1054	spread it out (v)	38	SPREAD OUT	
+Nagano-2013-1256-579	579	1056	hang it on (v)			
+Nagano-2013-1256-580	580	1058	leash (v)	2959	LEASH	
+Nagano-2013-1256-581	581	1060	put it in (v)			
+Nagano-2013-1256-582	582	1061	put it in, pour (v)	1487	POUR	
+Nagano-2013-1256-583	583	1063	put it out (v)			
+Nagano-2013-1256-584	584	1065	collect (v)	7	GATHER	
+Nagano-2013-1256-585	585	1066	save, store up (v)			
+Nagano-2013-1256-586	586	1068	make (v)	734	MAKE	
+Nagano-2013-1256-587	587	1070	destroy it (v)	681	DESTROY	
+Nagano-2013-1256-588	588	1072	repair it (v)	2243	REPAIR	
+Nagano-2013-1256-589	589	1074	split (vi)	1437	SPLIT	
+Nagano-2013-1256-590	590	1075	split (vi)	1437	SPLIT	
+Nagano-2013-1256-591	591	1076	tear (vt)	1735	TEAR (SHRED)	
+Nagano-2013-1256-592	592	1078	bend it (v)	2004	BEND	
+Nagano-2013-1256-593	593	1080	fold (v)	1470	FOLD	
+Nagano-2013-1256-594	594	1081	fold (v)	1470	FOLD	
+Nagano-2013-1256-595	595	1083	wash it (v)	1453	WASH	
+Nagano-2013-1256-596	596	1085	roll it (v)	827	ROLL	
+Nagano-2013-1256-597	597	1086	wind it (v)	1113	WRAP	
+Nagano-2013-1256-598	598	1088	tie it (v)	1917	TIE	
+Nagano-2013-1256-599	599	1091	untie it (v)	352	UNTIE	
+Nagano-2013-1256-600	600	1092	untie it (v)	352	UNTIE	
+Nagano-2013-1256-601	601	1093	come loose (vi)			
+Nagano-2013-1256-602	602	1094	come loose (vi)			
```

Comment:
This is doubled - is this correct? Different source lexemes?

Reply:
Yes, it is correct because they distinguish cognate sets with the same meaning (so that for each language the value was placed under either of the glosses according to a non specified judgment by the authors, but as far as I remember that was not one case when a language had both).

This happened during data collection, apparently, but unfortunately it is not consistent, so we cannot use it as a source for cognate judgment. It might have the practice of a single fieldworker, but we don't have enough information for that either... :disappointed: